i18n/src/closureI18nExtractor.js

```diff
@@ -58,10 +60,12 @@ function extractCurrencySymbols(content) {
 
 function extractDateTimeSymbols(content, localeInfo) {
   //eval script in the current context so that we get access to all the symbols
+  // eslint-disable-next-line no-eval
   eval(content.toString());
   for (var propName in goog.i18n) {
     var localeID = findLocaleId(propName, 'datetime');
     if (localeID) {
+      // eslint-disable-next-line no-unused-vars
```

Comment:
>
> In i18n/src/closureI18nExtractor.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72910295>:
>
> >    eval(content.toString());
> >    for (var propName in goog.i18n) {
> >      var localeID = findLocaleId(propName, 'datetime');
> >      if (localeID) {
> > +      // eslint-disable-next-line no-unused-vars
>
> Instead of this, you could change the
> localeInfo[localeID].DATETIME_FORMATS... to info.DATETIME_FORMATS....
>
Nice catch!